src/main/java/tom/TaskList.java

```diff
@@ -0,0 +1,56 @@
+package tom;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Represents the list of tasks to be done by the user.
+ */
+public class TaskList {
+    private ArrayList<Task> ls;
```

Comment:
Should this be renamed as `tasks` and of type `List<Task>` for better flexibility?
```suggestion
    private List<Task> tasks;
```